client/Styles/Header.css

```diff
@@ -0,0 +1,16 @@
+.component-header {
+    padding: 15px;
+    position: relative;
+    font-weight: normal;
+    font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 32px;
+    text-align: center;
+    color: #333;
+  }
+  
```

Comment:
Watch your indentation on this file. Looks like lines 2-end is indented further than they need to be.